web/styles/popovers.css

```diff
@@ -1431,7 +1433,11 @@ ul.popover-menu-list {
 #theme-switcher,
 .sidebar-topic-visibility-switcher {
     margin: 2px 10px;
+}
 
+#theme-switcher,
+.sidebar-topic-visibility-switcher,
+.recipient-bar-topic-visibility-switcher {
```

Comment:
These blocks could likely be cleaned up by adding classes with names like `tab-option-group-horizontal` and `tab-option-group-vertical`, if that's what differs between the two categories. Not sure whether `tab-option-group` is the right prefix, but seems reasonable?

Reply:
Had another look at `popovers.css`. I realised that `.tab-option-content:focus-visible` is actually required for all the tab pickers in the popovers, so I moved to `.popover-menu > .tab-picker` to now apply to any tab picker that resides inside a popover.

Reply:
The `margin: 2px 10px;` property was required for those tab pickers which are placed alongside other popover menu options, such as the theme switcher in the personal menu or the topic visibility switcher inside the topic actions menu. So, incorporating your suggestion of the `tab-option-group` prefix, and the general naming convention in use across the popovers (`.popover-menu-*`), I ended up with `popover-menu-tab-group`.  How does that sound?

Reply:
`popover-menu-tab-group` seems fine.